content/blog/2018-02-10-drake.md

```diff
@@ -0,0 +1,117 @@
+---
+slug: "drake"
+title: "The prequel to the drake R package"
+package_version: 5.0.0
+authors:
+  - name: Will Landau
+    url: https://github.com/wlandau
+date: 2018-02-10
```

Comment:
Leave date as is for now. Tentative publication date is 2018-02-13, but this may be postponed by ~ a week if a priority rOpenSci post must be published - e.g. announcing unconference nominations open